### ISSUE TITLE
[8.1+] Scrolling through the viewport using touch is virtually impossible on Android.

### ISSUE THREAD

**Issue by hoshito-yurei** [renpy/renpy#4838]
https://github.com/renpy/renpy/assets/139383397/f165bfba-bf38-4379-80c8-95b85593ad1c

https://github.com/renpy/renpy/assets/139383397/cefde9d6-badd-4a4e-8122-6b597127d5e2

Firstly, I have attached two test videos for reference. The devices used for testing were the Motorola Edge 30 (Snapdragon 778G+, Android 13, XT2203-1) and the Samsung Galaxy S20 (Snapdragon 865, Android 13, SM-G981N). I have included visualized touch points in the videos.

The first video is from my game, and the second one is from a new Ren'Py sample project. In my game, proper scrolling is not possible, but it works correctly in the sample project.

I understand that Ren'Py 8.1 update added acceleration effects to scrolling via touch, making it smoother. It was something I had been looking forward to. However, it seems to have unintentionally affected the behavior. In my previous project before this update, scrolling via touch worked fine, but after the update, scrolling became extremely stuttering, and most touches are ignored, making it practically impossible (though it is possible if you scroll very delicately).

Considering that it works correctly in the sample project, it appears that this issue occurs as the project size increases, the content within the viewport becomes longer, and more elements are displayed around it. However, in my project, even though the dialogue doesn't increase significantly (around 40 lines), the issue still occurs. Hence, it doesn't seem to be a problem specifically related to the length of the viewport in my project, and it occurs regardless of the presence of predict in the screen. Therefore, it can be considered a relatively common issue. It is possible that the presence of imagebutton elements within the viewport could be the issue.
Another important point to note is that the issue occurs when scrolling by directly touching the viewport, but it does not occur when dragging the scrollbar.

To assist with resolving this problem, I have sent my project file to '`pytom@bishoujo.us`' along with a brief explanation and reproduction steps. I kindly request that you check your email for further details.

Thank you very much for always making efforts to solve problems.

**Comment by Org-gith** [comment]
I have noticed the same behavior in my project, basically perfect on paper, but once history is pretty filled with text in a heavy project, the acceleration drag is very laggy, whereas the old behavior doesn't show such signs (I had to revert the commit). I think it works well if the game is light/small, but once a game has a certain size or is not the most optimized, it gets laggy. 
I thought it was affecting me only so I didn't mention it, maybe a config could solve it.
